Makefile

```diff
@@ -893,8 +922,8 @@ endif  ## end of checking for WSL
 
 
 ### Creates a bootable USB drive that can be inserted into a real PC based on the compiled .iso. 
-boot : export override THESEUS_CONFIG += mirror_log_to_vga
-boot: check-usb $(iso)
+usb : export override THESEUS_CONFIG += mirror_log_to_vga
+usb: check-usb $(iso)
```

Comment:
There might be a conflict with having a target called `usb` _and_ an env var called `usb`. Even if it works, it's a bit confusing. 

We can change the environment variable to `drive` or something instead of `usb` (and document it as such).

Reply:
done